packages/wrangler/src/pages.tsx

```diff
@@ -1,17 +1,22 @@
 /* eslint-disable no-shadow */
 
+import assert from "assert";
 import type { BuilderCallback } from "yargs";
 import { join } from "path";
 import { tmpdir } from "os";
 import { existsSync, lstatSync, readFileSync } from "fs";
 import { execSync, spawn } from "child_process";
-import { Headers, Request, Response } from "undici";
-import type { MiniflareOptions } from "miniflare";
-import type { RequestInfo, RequestInit } from "undici";
+import { URL } from "url";
```

Comment:
Don't think this is required. URL should be available in Node's global scope since v10 (https://nodejs.org/api/globals.html#url).